misc/estimate_contract_call.test.js

```diff
@@ -0,0 +1,65 @@
+const { TezosToolkit, MichelsonMap } = require("@taquito/taquito");
```

Comment:
the files in the `misc` folder are just a bunch of scripts I use for testing various aspects of the contracts. This is not production code, just test snippets